sdk/lusid/utilities/api_client_builder.py

```diff
@@ -52,6 +52,8 @@ def build(cls, api_secrets_filename=None, id_provider_response_handler=None, api
         # Load the configuration
```

Comment:
You need to add `flag_to_use_decimal` as an argument to `build` above.

Reply:
I suggest following the same pattern as the `tcp_keep_alive` parameter and just defaulting it to `False` in the parameters.